README.md

```diff
@@ -116,6 +116,29 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system is configured to run Insights remediation; valid values are
 `present` (to ensure remediation) and `absent`.
 
+    rhc_insights:
+      tags: None
+
+A dictionary of tags that is added to the system record in Host Based Inventory (HBI).
+Used to search for hosts. This tags has keys and values. Values for each key can be any 
+data type. Posible values of `tags` parameter:
+- `tags` is null does not change the tags file content.
+- `tags` is {} remove all the tags by removing the file.
+- `tags` is non-empty, create the file with the specified tags
```

Comment:
The problem with this is that in other similar cases in this role, the use of `state: absent` is used to remove something.  Now the user will have to remember "In case of registration and proxy, I have to use `state: absent`, but in the case of tags, I have to use `{}` to do the same thing."

Reply:
Oops, you are right, and I even mentioned `{"state":"absent"}` in my first comment... :-(

Alba: sorry for the wrong suggestion today, let's adopt the same style of `{"state":"absent"}` for removing tags.

Reply:
Ok, this makes the role more robust.